src/main.js

```diff
@@ -1,29 +1,58 @@
 import { Home } from "../node_modules/genie/src/components/home.js";
 import { Loadscreen } from "../node_modules/genie/src/components/loadscreen.js";
-import { GameTest } from "../node_modules/genie/src/components/test-harness/test-screens/game.js";
-import { ResultsTest } from "../node_modules/genie/src/components/test-harness/test-screens/results.js";
+import { Select } from "../node_modules/genie/src/components/select.js";
+import { BalloonPumpGame } from "./components/balloon-pump-game.js";
+import { Results } from "../node_modules/genie/src/components/results.js";
 import { startup } from "../node_modules/genie/src/core/startup.js";
 
 const transitions = [
     {
         name: "loadscreen",
         state: new Loadscreen(),
         nextScreenName: () => "home",
+        // nextScreenName: () => "game",
```

Comment:
Can this be removed now?

Reply:
I left it in as it makes it easy when you're developing a game to shortcut straight to the game itself. But it's not super important we leave it in.